docs/contribute-module.md

```diff
@@ -40,78 +36,38 @@ Fork either the repo that was just created or the module repo that you want to u
+1.  Update the `README.md` file in the same `examples` directories to provide some basic information about what each example does.
 
 ### About end-to-end examples
 
 [inc-examples](inc-examples.md ':include')
 
-## Run examples and clean up
-
-[inc-examples-run](inc-examples-run.md ':include')
-
-## Update the PR tests
-
-Now that you tested the module by running your examples directly, update the `pr_test.go` file to run the examples as a unit test. This file runs as part of pull request pipelines. For more information about testing modules in this project, see [Tests](tests.md).
-
-[tests.md](inc-tests-create.md ':include')
-
-## Run the PR tests
-
-[example-test](inc-example-test.md ':include')
-
-For more information about testing modules in this project, see [Tests](tests.md).
-
 ## Push your code and submit for review
 
 After your PR tests pass, create a pull request.
 
-### Verify changes
-
-[inc-precommit](inc-precommit.md ':include')
-
 ### Open a pull request
 
-1.  Open a pull request:
+1.  Start by contributing a draft PR. A draft pull request provides insight about the work that you're doing but cannot be merged. To create a draft pull request, use the drop-down and select **Create Draft Pull Request**, then click **Draft Pull Request**.
+    - Open your PR from your working branch to `main`.
+    - Provide a clear title and description. Be sure to merge the latest changes from upstream before you create the PR.
 
-    1.  Start by contributing a draft PR. A draft pull request provides insight about the work that you're doing but cannot be merged. To create a draft pull request, use the drop-down and select **Create Draft Pull Request**, then click **Draft Pull Request**.
-        - Open your PR from your working branch to `main`.
-        - Provide a clear title and description. Be sure to merge the latest changes from upstream before you create the PR.
-    1.  When you're ready to merge your code, mark your draft pull request as ready for review. For more information, see [Changing the stage of a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request).
-    1.  If you are in the `github-collaborators` team, you can run CI pipeline workflow by adding a comment to the pull request with the following text:
+### Request a code review
 
-        ```text
-        /run pipeline
-        ```
+When you're ready to merge your code, request a review. Your PR is reviewed by the module maintainers and the community.
 
-      ?> **Retriction:** If the CI pipeline workflow doesn't run when you post the comment, you don't have access to run the pipeline. The merging team will run the workflow for you.
+1.  Add a `Ready for review` comment to the PR. The comment triggers automation that adds the `ready-for-review` label to your PR.
```

Comment:
This needs to be done after the PR is out of draft mode. Pipelines do not run against draft PRs